### ISSUE TITLE
Small error in CDK Workshop Code?

### ISSUE THREAD

**Issue by RajivKYadav** [aws-samples/aws-cdk-intro-workshop#70]
### Environment

  - **CDK CLI Version:** 
  - **Section:** 
  - **Browser:** 
  - **Language:**

**Comment by Jerry-AWS** [comment]
@RajivKYadav Thanks for submitting this over here. (For readers: this was originally a bug in the CDK Guide repo. See https://github.com/awsdocs/aws-cdk-guide/issues/105)

**Comment by Jerry-AWS** [comment]
@RajivKYadav I can't reproduce this problem. I've run through the CDK Workshop again and I did run into a problem at the same spot, but it turned out I had some stale package (in particular, my DynamoDB Table Viewer wouldn't upgrade to the latest version because I had a handful of modules stuck at 0.3.8). Once I corrected this, the code in the Workshop worked without modification.

Try deleting your CDK package and reinstalling. On Windows this is:

```
rmdir /s /q %appdata%\npm\node_modules\aws-cdk
npm install -g aws-cdk
```
Then update everything:

```
npm update
npm update -g
```

I also had to stop my `npm run watch` window and restart it.

If you can, give it another try. I don't think there's actually a problem with this step.

**Comment by RajivKYadav** [comment]
HI Jerry,
Thanks for the follow up.  I'll give this a try over the weekend and get
back to you.  What you are saying would make sense as it was the viewer
that was giving an error and looks like that might have been fixed in the
updated version of the viewer.

Best Regards,
Rajiv
(508)817-6147

**Comment by javydekoning** [comment]
This was fixed in #84 , can we close this @eladb ?
